packages/components/src/__future__/ExampleComponent/ExampleComponent.tsx

```diff
@@ -3,7 +3,7 @@ import styles from "./ExampleComponent.module.scss"
 
 // Placeholder for illustration DO NOT COPY
 export const ExampleComponent = (): JSX.Element => (
-  <div className={`${styles.exampleComponent} bg-purple-600`}>
+  <div className={`${styles.exampleComponent} bg-green-300`}>
```

Comment:
Haha why?

Reply:
Page Templates was already using `bg-purple-600` so it was in their CSS bundle and I needed to test whether or not our tailwind styles were making it through, and the component was just working cause the consumer had the class already 😄